docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -46,14 +46,27 @@ image::images/add-data-fleet.png[Add data using Fleet]
 [[upload-data-kibana]]
 === Upload a file
 
-experimental[] If your data is in a CSV, JSON, or log file, you can upload it using the {file-data-viz}. You can upload a file up to 100 MB. This value is configurable up to 1 GB in
+If your data is in a CSV, JSON, or log file, you can upload it using the {file-data-viz}. You can upload a file up to 100 MB. This value is configurable up to 1 GB in
 <<kibana-ml-settings, Advanced Settings>>. To upload a file with geospatial data,
 refer to <<import-geospatial-data, Import geospatial data>>.
 
 [role="screenshot"]
-image::images/add-data-fv.png[File Data Visualizer]
-
-
+image::images/add-data-fv.png[{file-data-viz}]
+
+The {stack-security-features} provide roles and privileges that control which
+users can upload files. To upload files in {kib}, you must have the following
+privileges:
+
+* `all` {kib} privileges for *Discover* or *{ml-app}* in the appropriate spaces.
+Alternatively, `read` {kib} privileges for *{ml-app}* and `all` {kib}
+privileges for *{ipm-app}*
+* `manage_pipeline` or `manage_ingest_pipelines` cluster privilege
+* `create`, `create_index`, and `read` index privileges for the destination
```

Comment:
`manage` is also needed for the index already exists check that happens when typing in the index name in the UI.

![image](https://user-images.githubusercontent.com/22172091/128328690-ccfd9592-9c7c-47c5-bcc6-a2c38b142859.png)

Reply:
This is a recent change, it now uses elasticsearch's [index exists api](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-exists.html) rather than the field caps api

Reply:
Thanks!  Added "manage" in 3cb953d56b742024debb48a14bf878b7e50246de